src/components/Login/LoginMobileNumberCard.tsx

```diff
@@ -54,16 +51,11 @@ export const LoginMobileNumberCard: FunctionComponent<LoginMobileNumberCard> = (
 
   const onRequestOTP = async (): Promise<void> => {
     setIsLoading(true);
-    try {
-      const fullNumber = createFullNumber(countryCode, mobileNumberValue);
-      await requestOTP(fullNumber, codeKey, endpoint);
-      setIsLoading(false);
-      setMobileNumber(fullNumber);
-      setLoginStage("OTP");
-    } catch (e) {
-      setIsLoading(false);
-      alert(e);
-    }
+    const fullNumber = createFullNumber(countryCode, mobileNumberValue);
+    const isRequestSuccessful = await handleRequestOTP(fullNumber);
+    setMobileNumber(fullNumber);
```

Comment:
Do we still want to set mobile number if the request of OTP fails? What are the advantages of using this true/false approach as compared to the try/catch approach?

Reply:
i was following the previous code when i did this, but yeah i don't think we should if the request fails.
i think this approach moves all the reusable error handling logic to the `LoginContainer`